lms/djangoapps/course_blocks/api.py

```diff
@@ -28,13 +30,17 @@ def get_course_block_access_transformers():
     """
     course_block_access_transformers = [
         library_content.ContentLibraryTransformer(),
-        start_date.StartDateTransformer(),
+        user_partitions_date_modificator.UserDatePartitionTransformer(),
         user_partitions.UserPartitionTransformer(),
+        # start_date.StartDateTransformer(),
```

Comment:
Please switch off this transformer in the initial line if we want to have this line in the code